src/redux/store.ts

```diff
@@ -0,0 +1,23 @@
+import { createStore, Store, combineReducers, applyMiddleware } from "redux";
+import { createWrapper, Context } from "next-redux-wrapper";
+import createSagaMiddleware, { Task } from "redux-saga";
+import { rootSaga } from "./sagas";
+import { rootReducer } from "./reducers";
+
+export interface State {}
+
+export interface SagaStore extends Store {
+  sagaTask?: Task;
+}
+
+export const makeStore = (context: Context) => {
+  const sagaMiddleware = createSagaMiddleware();
+
+  const store = createStore(rootReducer, applyMiddleware(sagaMiddleware));
```

Comment:
`createStore` is deprecated. Can we switch to something else?

Reply:
Updated to use redux toolkit. Replaced with `configureStore`.